jvcl/run/JvRichEdit.pas

```diff
@@ -51,6 +51,14 @@ interface
   Forms, Graphics, StdCtrls, Dialogs, RichEdit, Menus, ComCtrls, SyncObjs,
   JvExStdCtrls, JvTypes;
 
+const
+  {$EXTERNALSYM MSFTEDIT_CLASSA}
+  MSFTEDIT_CLASSA       = 'RICHEDIT50A';     { Richedit4.0 Window Class. }
+  {$EXTERNALSYM MSFTEDIT_CLASSW}
+  MSFTEDIT_CLASSW       = 'RICHEDIT50W';     { Richedit4.0 Window Class. }
+  {$EXTERNALSYM MSFTEDIT_CLASS}
+  MSFTEDIT_CLASS = MSFTEDIT_CLASSW;
```

Comment:
Instinctively, I believe this should be IFDED depending on whether the RTL is unicode or not. However, looking at line 2840, I even wonder if it shouldn't simply be the ANSI version

Reply:
[RichEdit 1.0](https://docs.microsoft.com/en-us/windows/win32/controls/about-rich-edit-controls#rich-edit-version-10) only supported ANSI. Starting with [RichEdit 2.0](https://docs.microsoft.com/en-us/windows/win32/controls/about-rich-edit-controls#rich-edit-version-20) Unicode was added.